dlp/snippets/src/test/java/dlp/snippets/DeIdentificationTests.java

```diff
@@ -549,4 +549,12 @@ public void testDeIdentifyWithDeterministicEncryption() throws IOException {
     String output = bout.toString();
     assertThat(output).contains("Text after de-identification:");
   }
+
+  @Test
+  public void testDeIdentifyDataReplaceWithDictionary() throws IOException {
+    DeIdentifyDataReplaceWithDictionary.deidentifyDataReplaceWithDictionary(
+        PROJECT_ID, "My name is Alicia Abernathy, and my email address is aabernathy@example.com.");
+    String output = bout.toString();
+    assertThat(output).contains("Text after de-identification:");
```

Comment:
Let's test the complete output to demonstrate what's happening better.

Based on the word list, this can only be one of two values, so something like
```
assertThat(output).isIn(ImmutableList.of(
  "Text after de-identification: My name is Alicia Abernathy, and my email address is izumi@example.com.",
  "Text after de-identification: My name is Alicia Abernathy, and my email address is alex@example.com."
)));
```